packages/actions/log/src/components/Log.js

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types'
+import {useEffect, useState} from 'react'
+import {FormattedMessage} from 'react-intl'
+import {EditableValue, Button, LoadMask, StatedValue} from 'tocco-ui'
+
+import {
+  StyledContainer,
+  StyledLogWrapper,
+  StyledHeader,
+  StyledLogFileStatedValue,
+  StyledFileCountStatedValue,
+  StyledButtonItem,
+  StyledTextAreaItem,
+  StyledTextarea
+} from './StyledComponents'
+
+const Log = ({data, fetchData, fileContent, fetchFileContent, intl}) => {
+  const [lines, setLines] = useState(500)
+  const [currentFile, setCurrentFile] = useState({key: 'nice.log', display: 'nice.log'})
```

Comment:
question(blocking): are you sure that there will be always an entry with key `nice.log`?  

if not, we should be able to set undefined as default and set the first file as soon as the data is loaded

Reply:
Jedes System hat ein nice.log